apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -18,6 +19,49 @@ type ResearchOutputParameters = {
   researchOutputId: string;
 };
 
+type ResearchOutputFetchOptions = FetchOptions & {
+  status: string;
+  associationId: string;
+};
+
+const researchOutputFetchOptionsValidationSchema: JSONSchemaType<ResearchOutputFetchOptions> =
+  {
+    type: 'object',
+    properties: {
+      take: { type: 'number', nullable: true },
+      skip: { type: 'number', nullable: true },
+      search: { type: 'string', nullable: true },
+      filter: {
+        type: 'array',
+        items: { type: 'string' },
+        nullable: true,
+      },
+      status: { type: 'string', const: 'draft' },
+      associationId: { type: 'string' },
+    },
+    if: {
```

Comment:
although I told you that you COULD do these conditionally I actually stand by what I said earlier which is that these should all be completely independent filters. I don't see why we can't just expose a teamId or workingGroupId filters. Similarily why can't we not just have a `status` filter?